examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/switch/.content.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          jcr:primaryType="cq:Page">
+    <jcr:content
+            cq:lastModified="{Date}2020-07-15T13:25:39.732+02:00"
+            cq:lastModifiedBy="admin"
+            cq:tags="[]"
+            cq:template="/conf/core-components-examples/settings/wcm/templates/content-page"
+            jcr:description="Switch"
+            jcr:primaryType="cq:PageContent"
+            jcr:title="Switch"
+            sling:resourceType="forms-components-examples/components/page">
+        <root
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="wcm/foundation/components/responsivegrid">
+            <responsivegrid
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="wcm/foundation/components/responsivegrid">
+                <text_1740702241
+                        cq:styleIds="[1644862132301]"
+                        jcr:created="{Date}2019-02-15T19:05:53.032+01:00"
+                        jcr:createdBy="admin"
+                        jcr:lastModified="{Date}2019-02-15T19:09:39.787+01:00"
+                        jcr:lastModifiedBy="admin"
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="core/wcm/components/text/v2/text"
+                        text="&lt;h1>Switch&lt;sub>v1&lt;/sub>&lt;/h1>"
+                        textIsRich="true"/>
+                <text
+                        cq:styleIds="[1544762734201]"
+                        jcr:created="{Date}2018-12-06T19:11:23.947+01:00"
+                        jcr:createdBy="admin"
+                        jcr:lastModified="{Date}2020-07-15T13:25:39.729+02:00"
+                        jcr:lastModifiedBy="admin"
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="core/wcm/components/text/v2/text"
+                        text="&lt;p>A switch in an Adaptive Form allows users to select one of two options. Each switch is represented by a Data Value (value used to process items of a switch) and display value (value to be sent to backend)&lt;/p>"
```

Comment:
Display value is not sent to the backend - it is the data value which is sent. Please correct.